.travis.yml

```diff
@@ -5,9 +5,9 @@ cache:
 jdk: openjdk11
 env:
   global:
-  # Sonatype jira info
-  - secure: nawOdRhJ1DX86SZhcS5JeHncU049DVvlG3m7OHkHrldwbL+jOIHCzjl7wBL0LnqdwqFXbntfPLnigLQFZ1eFNgXVesZKDQbyzazGF3nZoJWSIGtuAt0GbORxyzW96TwowE1AP4og0FLpdCCwQJ1cRjTiaPnZFl6yJ2gwdaQBmf50IH+WrEWvkphd2feB8uqSv0GgKtVCwJnBwhHrPQrE2DAuA98lSDGhWJYASdRRVJ8NXJ+1CtvCTUacaEiiRev7JK15Wz7HS3GGyDdcxg9mBoYFs2HhOn0nXMiZqa66VbVSJWGroCve4qngri7DnBzeeJUguuBKQzHOYtfbyNgxyxlQrT6PKxs7YBcfMCcszR6vTlI3qb6tsMS6BWyLvGOgJNRoGa1iw/+Uc1hKBOf+Wtq4q4EeJ27c6i4pv4aVCTEP26rvT/CYXi05wTYkH0PtZaPuKpCqZgp+kIAo2LpdrhMXf6QbsBCWP0431T7r+d79f//D/IBr4S00iQBukPxU4shujCdbsKIet7hBcAhjbGUkx1odMNiaSn2JTs3KJTYq2adK5zC5ufkK+rTDCAgaYeuMIgZwuSopNpXTAV17OavJQ2v+iP/3+lINpKI7kHuT3qYF0rFyTX0ZBApZE9XMscozuLWKW3aIlqvuiPPULPY1mZHBPBHc/LPG3GlNXeg=
-  - secure: OqOKGAvbqC1uQurrPAVknHxSZKzgcXoKPXvgJMPcito2PFe3jRZuCXWWDAPnEPNjtH/O/jC37Ne07VQBq3KJ3d83WPniC+d6MXwsh4cP9tvIGT0cwAB+iEv+BkAkTHAGVDqdDYUJCJfvq+XMDujpR7ijNKt9HbSu3BPMUZlNovmY7WL+4Ei+KTj2bi/5jb8SkBKqGEpvL245r5w9+/NUPY7ifY0F+c8elvtJ44U65BSvBBw9LFrzYvcS812e1L5iRswTDGbqcXCaqVsP21S+SsjXBmtFsy2IVutmGnTEfLSZ5Y9JbR96ux/49YfBp3FRDwqIwztm4Y4cTij1J5dC/bpCAElOPhFaRk/luGHr44Sz90ZxqsqDH1XSmjDNNlpxUHSg9bhuZRqdseomokFHR9mGKxGBQYe5+MyciyGM/9AmTW87FdqNuNKNpJvhBuaPoJR1IqVmCgPYfOEyKHbPNapr42NgrmnXnDQvGjCcuEbk1WSTHe0Mp8n+0FVlk2cRSwd5juniOz4k9FKeTp8CCqQz3XR+x811z7HUMROSg6IKIA/XC35HIqfAGYfqUXktWobGiTbyngoBE4feTqtCHtoZAEcuzlWtGmTlVGm4ELJJ8pf6hhjWCihFeY62XNAcPeAUg8o5WE9MQV32rjCHVE2AW0iV6UWRh8XJwUFCX/I=
+  # EG OSS Sonatype jira info
```

Comment:
Should be HCom OSS to avoid confusion with the EG OSS credentials we have for publishing to ExpediaGroup org.

Reply:
Ok, done